tests/installed/nondestructive/itest-pull.sh

```diff
@@ -30,6 +30,20 @@ ostree --repo=bare-repo init --mode=bare-user
 ostree --repo=bare-repo remote add origin --set=gpg-verify=false $(cat ${test_tmpdir}/httpd-address)
 log_timestamps ostree --repo=bare-repo pull --disable-static-deltas origin ${host_nonremoteref}
 
+echo "ok pull"
```

Comment:
Don't you need to update some number-of-tests count when you add a test?

Reply:
These tests aren't yet being parsed by something that understands TAP...someday later.